src/main/telemetry/crsf.c

```diff
@@ -140,25 +140,29 @@ bool bufferCrsfMspFrame(uint8_t *frameStart, int frameLength)
 
 bool handleCrsfMspFrameBuffer(uint8_t payloadSize, mspResponseFnPtr responseFn)
 {
-    bool requestHandled = false;
+    static bool replyPending = false;
+    if (replyPending) {
+        replyPending = sendMspReply(payloadSize, responseFn);
+        return replyPending;
+    }
     if (!mspRxBuffer.len) {
```

Comment:
Wondering if this check should be done _before_ we are trying to send another chunk?
Also, returning `replyPending` is cleaner here.

Reply:
Then the rest of the reply wouldn't be sent until the buffer starts filling up with the next request. The idea here is to give sending the reply the highest priority to just get it done as fast as possible. We could return ```replyPending``` as it will be false and it should return false here. 
I think we could make this more readable and robust by separating the handling of buffer data and sending of the reply. Right now the ```mspReplyPending``` can mean a lot of things.

Reply:
Yeah, refactoring this would probably help - it is hard to understand as it is. We can merge this as a fix of the actual problem, and keep refactoring as something to do against `master` at a later stage.

Reply:
Great! I'm looking into it. Will follow up with another PR soon 👍